tests/Foundatio.Tests/Jobs/WorkItemJobTests.cs

```diff
@@ -246,7 +246,7 @@ public class WorkItemJobTests : TestWithLoggingBase {
 
             await job.RunUntilEmptyAsync();
             await countdown.WaitAsync(TimeSpan.FromSeconds(2));
-            Assert.Equal(0, countdown.CurrentCount);
```

Comment:
We should keep this assert at 0 and update the `var countdown = new AsyncCountdownEvent(2);`  to a value of 2.

Reply:
done